src/Idris/CommandLine.idr

```diff
@@ -205,7 +207,9 @@ options = [MkOpt ["--check", "-c"] [] [CheckOnly]
               (Just "Generate profile data when compiling, if supported"),
 
            optSeparator,
-           MkOpt ["--no-shadowing-warning"] [] [IgnoreShadowingWarnings]
+           MkOpt ["-Werror"] [] [WarningsAsErrors]
+              (Just "Treat warnings as errors"),
+           MkOpt ["-Wno-shadowing"] [] [IgnoreShadowingWarnings]
```

Comment:
renamed this flag for consistency (where -Wshadowing would be to enable shadow warnings in the future)

Reply:
You'll have to update the `.ipkg` files for `base` & so on then.

Reply:
Renamed all usages on this repo